pybamm/models/submodels/electrode.py

```diff
@@ -34,7 +34,10 @@ def __init__(self, phi_s, eps, j, param):
 
         # different bounday conditions in each electrode
         if phi_s.domain == ["negative electrode"]:
-            i_s_n = -param.sigma_n * (1 - eps) ** param.b * pybamm.grad(phi_s)
+            if eps is None:
```

Comment:
might be good to explain this line (and other similar ones) with a comment, e.g. "if no porosity eps is provided, the model defaults to using the parameter epsilon_n"
EDIT: I see you've done this below, would still be good to repeat here in case someone is just looking at this function